src/components/split-button/split-button.spec.js

```diff
@@ -621,6 +533,63 @@ describe('SplitButton', () => {
       wrapper.unmount();
     });
   });
+
+  describe('when the theme is set to Classic', () => {
+    it('renders styles correctly', () => {
+      wrapper = renderWithTheme({ carbonTheme: classicTheme }, singleButton, TestRenderer.create);
+      expect(wrapper).toMatchSnapshot();
+    });
+
+    it('renders Toggle Button left border as expected', () => {
+      const mockProps = { carbonTheme: classicTheme, buttonType: 'primary' };
+
+      wrapper = renderWithTheme(mockProps, singleButton, mount);
+
+      assertStyleMatch({
+        borderLeft: '1px solid #1e499f'
+      }, wrapper.find(StyledSplitButtonToggle));
+    });
+
+    it('applies the expected styles to the toggle when disabled is false and the displayed prop is true', () => {
+      wrapper = renderWithTheme({ carbonTheme: classicTheme }, singleButton, mount);
+      simulateFocusOnToggle(wrapper);
+
+      assertStyleMatch({
+        backgroundColor: '#1963f6',
+        borderColor: '#1963f6'
+      }, wrapper.find(StyledSplitButtonToggle), { modifier: '&:active' });
+
+      assertStyleMatch({
+        backgroundColor: '#1e499f',
+        borderColor: '#1e499f'
+      }, wrapper.find(StyledSplitButtonToggle), { modifier: '&&' });
+    });
+
+    describe('and the Split Button Menu is open', () => {
+      beforeEach(() => {
+        wrapper = renderWithTheme({ carbonTheme: classicTheme }, singleButton, mount);
+        simulateFocusOnToggle(wrapper);
+      });
+
+      it('it\'s buttons should have the expected border', () => {
+        assertStyleMatch({
+          backgroundColor: '#1e499f',
+          border: '1px solid #1e499f'
+        }, wrapper.find(StyledSplitButtonChildrenContainer), { modifier: `${StyledButton}` });
+      });
+
+      it('it\'s buttons shoul have the expected background when focused', () => {
```

Comment:
```suggestion
      it('its buttons should have the expected background when focused', () => {
```

Reply:
changed

Reply:
Thanks for fixing this: https://github.com/Sage/carbon/commit/8e55c3204b58536aec3fc01bbea09a3be48cff0c#diff-c73ad659f90eb4d633ceff2950c3bdce
```diff
- it\'s
+ its
```
Please could you also fix this?
```diff
- shoul
+ should
```

Reply:
done :)

Reply:
Thanks :slightly_smiling_face: :white_check_mark: 75f494a